src/eventhandling-new.ts

```diff
@@ -0,0 +1,82 @@
+const log = require('./log');
+
+// wew use the mixin approach as described here:
+// https://mariusschulz.com/blog/mixin-classes-in-typescript
+type Constructor<T = {}> = new (...args: any[]) => T;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type EventHandler = (event?: any) => void;
```

Comment:
I think all `any` should at least throw warnings, because TypeScript is rather useless with `any` definitions in the chain.

Reply:
It's unclear to me at this point how the events can look like, that's why I used any for now, but I agree that unknown is a better temporary solution.

I would suggest to spend some time when we converted all the files to typescript. There are a lot of declarations (e.g. the googledrive.ts) with no typescript type declared (even though they have jsdoc declarations). How would you feel about activating `noImplicitAny` flag in the tsconfig.json?

Reply:
> It's unclear to me at this point how the events can look like, that's why I used any for now, but I agree that unknown is a better temporary solution.

That's not what I meant. I meant it should throw a warning (instead of ignoring the error with this config), so we can later easily work down the linter warnings in the code. Maybe I misunderstood the code, and the warning is still intact with this comment?

Reply:
Ahhh. Sorry, I misunderstood you. I added a todo on top. I leave it at that for now.